api_client.go

```diff
@@ -515,6 +516,7 @@ func (c *APIClient) prepareRequest(
 
 	// Generate a new request
 	if body != nil {
+		log.Printf("[DEBUG] the request to nsx-t to register method: %s, url: %s, body: %s", method, url.String(), body)
```

Comment:
I am not sure if this log description is appropriate.

Reply:
forget to remove this debug message. remove it.